src/hooks/useActiveOrg.tsx

```diff
@@ -0,0 +1,61 @@
+import { getActiveOrgId, setActiveOrgId as setActiveOrgIdCookie } from "@propelauth/javascript"
+import { useContext, useEffect, useState } from "react"
+import { AuthContext } from "../AuthContext"
+
+const ORG_SELECTION_LOCAL_STORAGE_KEY = "__last_selected_org"
+const ACTIVE_ORG_ID_LOCAL_STORAGE_KEY = "__ACTIVE_ORG_ID"
+
+export function useActiveOrg() {
+    const context = useContext(AuthContext)
+    if (context === undefined) {
+        throw new Error("useActiveOrg must be used within an AuthProvider or RequiredAuthProvider")
+    }
+    const [activeOrgIdState, setActiveOrgIdState] = useState<string | undefined>(getActiveOrgId())
+
+    const handleLocalStorageChange = () => {
+        setActiveOrgIdState(localStorage.getItem(ACTIVE_ORG_ID_LOCAL_STORAGE_KEY) ?? undefined)
+    }
+
+    const setActiveOrgId = (orgId: string) => {
```

Comment:
One extra thing we can do here - if the user isn't in that org we shouldn't set the cookie and we should return an error/false.